package.json

```diff
@@ -7,6 +7,11 @@
     "build": "./node_modules/.bin/tsc",
     "lint": "./node_modules/.bin/eslint src/**/*.ts --fix"
   },
+  "lint-staged": {
+    "*.ts": ["eslint --fix", "git add"],
+    "*.{json,md}": ["prettier --write", "git add"],
+    "{.eslintrc}": ["prettier --write", "git add"]
+  },
```

Comment:
`.lintstagedrc` ?

Reply:
Can be useful, do you have ideas what files needed to add? @DenisDashkevich

Reply:
oh, got you))